Support for multiple distinct instances, as in Passport itself (#186)
fiznool/passport-oauth2-refresh: CHANGELOG.md

All notable changes to this project will be documented in this file.
This project adheres to [Semantic Versioning](http://semver.org/).

> ## [2.2.0] - 2023-04-06

Reviewer: Let's change this to `Unreleased` as we don't have a definitive version number or release date yet.

```suggestion
## [Unreleased]
```

### Added

> - In addition to exporting an object that can be immediately used,
>   just as before, we have added an `AuthTokenRefresh` property. This is a
>   constructor that can be invoked to create a distinct instance, for
>   applications that require more than one passport instance. (Passport has the same feature.)

Reviewer: I think we can make this more concise. How about the following:

```suggestion
- Added a new named export `AuthTokenRefresh`. This is a
  constructor that can be invoked to create a distinct instance, for
  applications that require more than one Passport instance.
```
